pkg/daemon/imagepuller/utils.go

```diff
@@ -96,20 +96,20 @@ func newStatusUpdater(imagePullNodeClient clientalpha1.NodeImageInterface) *stat
 	}
 }
 
-func (su *statusUpdater) updateStatus(nodeImage *appsv1alpha1.NodeImage, newStatus *appsv1alpha1.NodeImageStatus) error {
+func (su *statusUpdater) updateStatus(nodeImage *appsv1alpha1.NodeImage, newStatus *appsv1alpha1.NodeImageStatus) (bool, error) {
```

Comment:
please name the return value or give some comment above the function about the return value